vm/src/stdlib/array.rs

```diff
@@ -584,6 +621,71 @@ impl PyArray {
         }
     }
 
+    #[pymethod]
+    fn fromunicode(zelf: PyRef<Self>, obj: PyObjectRef, vm: &VirtualMachine) -> PyResult<()> {
+        let str = PyStrRef::try_from_object(vm, obj.clone()).map_err(|_| {
```

Comment:
how about avoiding `str` not to be confused with Rust `str`?

Reply:
No problem.